README.md

```diff
@@ -63,6 +63,21 @@ kubectl create secret generic snyk-monitor -n snyk-monitor --from-file=./dockerc
 kubectl create configmap snyk-monitor-certs -n snyk-monitor --from-file=<path_to_certs_folder>
 ```
 
+5. If you are using an insecure registry or your registry is using unqualified images, you can provide `registries.conf` file. It is a system-wide configuration file for container image registries. The file format is TOML. For more information on how to format and examples visit: https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md 
+
+Create a file named `registries.conf`, see example adding an insecure registry: 
+
+```
+[[registry]]
+location = "localhost:5000"
```

Comment:
Not sure if it makes difference. But what do you think to include a comment saying the user should change it for her location
```
[[registry]]
location = "localhost:5000" # change for your location
insecure = true
```

Reply:
yeah I agree! I do think that should change, or just `<your-insecure-registry>` instead of `localhost`?

Reply:
I've changed to 'internal-registry-for-example.net/bar', it's an example on https://github.com/containers/image/blob/master/registries.conf, copied it 🙂